lib/dashboard/home/tab_bar/tokens/insert_withdrawal_amount/view/insert_withdrawal_amount_page.dart

```diff
@@ -6,34 +6,46 @@ import 'package:altme/withdrawal_tokens/withdrawal_tokens.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-class InsertWithdrawalAmountPage extends StatefulWidget {
+class InsertWithdrawalAmountPage extends StatelessWidget {
   const InsertWithdrawalAmountPage({
     Key? key,
     required this.withdrawalAddress,
   }) : super(key: key);
 
   final String withdrawalAddress;
 
-  static Route route({
-    required String withdrawalAddress,
-  }) {
+  static Route route({required String withdrawalAddress}) {
     return MaterialPageRoute<void>(
-      builder: (_) => BlocProvider<InsertWithdrawalPageCubit>(
-        create: (_) => InsertWithdrawalPageCubit(),
-        child: InsertWithdrawalAmountPage(
-          withdrawalAddress: withdrawalAddress,
-        ),
-      ),
+      builder: (_) =>
+          InsertWithdrawalAmountPage(withdrawalAddress: withdrawalAddress),
+      settings: const RouteSettings(name: '/insertWithdrawalAmountPage'),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<InsertWithdrawalPageCubit>(
+      create: (_) => InsertWithdrawalPageCubit(),
+      child: InsertWithdrawalAmountView(withdrawalAddress: withdrawalAddress),
     );
   }
+}
+
+class InsertWithdrawalAmountView extends StatefulWidget {
```

Comment:
@bibash28 Here, it's better to be a private class because not used outside this file.